fe/fe-core/src/main/java/com/starrocks/sql/parser/AstBuilder.java

```diff
@@ -752,7 +753,13 @@ public ParseNode visitTableName(StarRocksParser.TableNameContext context) {
             partitionNames = (PartitionNames) visit(context.partitionNames());
         }
 
-        TableRelation tableRelation = new TableRelation(tableName, partitionNames);
+        List<Long> tabletIds = Lists.newArrayList();
+        if (context.tabletList() != null) {
+            tabletIds = context.tabletList().INTEGER_VALUE().stream().map(ParseTree::getText)
```

Comment:
What if tablet list is empty?

Reply:
tabletList cannot be empty. In antlr, the "?" keyword will produce Null and non-NULL, and the "*" keyword is an empty List, or non-empty.